### DISCUSSION TITLE
Reorder Apps (i.e., CRUD Editors)

### DISCUSSION THREAD

**Discussion by relayer464** [TiagoSilvaPereira/vemto-issues#216] [Ideas & Suggestions]
Many of my projects have quite a few Apps (i.e., CRUD editors).  It appears that when you modify one, it goes to the bottom of the list?  I would like to suggest that there was a way to sort the order of the Apps.  Even if they just defaulted to alphabetical order I think it would make it easier to identify the particular App you want to work with.

Thanks!

**Comment by TiagoSilvaPereira** [comment]
Hi @relayer464 thank you for your suggestion
